lib/models/project.js

```diff
@@ -185,6 +189,10 @@ class Project {
     return !!this.pkg.keywords && this.pkg.keywords.indexOf('ember-addon') > -1;
   }
 
+  isModuleUnification() {
```

Comment:
is it supposed to be a public API? Maybe it worth to mark this method as private or protected in order to avoid following deprecation when `isModuleUnification` becomes default?

Reply:
Good point. I think it should be private as it's just a temporary internal API to allow us to branch the blueprints.

Reply:
updated